nucliadb/src/nucliadb/search/search/chat/ask.py

```diff
@@ -190,7 +190,7 @@ async def ndjson_stream(self) -> AsyncGenerator[str, None]:
 
     def _ndjson_encode(self, item: AskResponseItemType) -> str:
         result_item = AskResponseItem(item=item)
-        return result_item.model_dump_json(exclude_none=True) + "\n"
+        return result_item.model_dump_json(exclude_none=True, by_alias=True) + "\n"
```

Comment:
Not using `by_alias` made us return relations using `from_` instead of `from` in our ask response